src/convert.test.ts

```diff
@@ -1,6 +1,6 @@
 import { convertAmount } from './convert'
 
-const ONE_ETH = 1000000000000000000n
+const ONE_ETH = BigInt(1000000000000000000)
```

Comment:
I changed this to avoid complaints as BigInt literals are not supported on targets < ES2020. We could massage the config, or target higher, what do you think?

Reply:
Is there an environment where `BigInt` shipped without the literal notation? :thinking: